### DISCUSSION TITLE
Adapt ebirds to automatically show your region

### DISCUSSION THREAD

**Discussion by alexbelgium** [Nachtzuster/BirdNET-Pi#123] [General]
Use this code to modify the ebirds url to automatically show your region. The region must be in ISO-3166 format (https://github.com/lukes/ISO-3166-Countries-with-Regional-Codes/blob/master/all/all.csv). Not all regions are supported by ebirds, you should test first manually by appending your country/region code to the url such as `https://ebird.org/species/eurlin1/FR`

```
REGION="FR"
sed -i "s|ebird?siteLanguage|ebird/$REGION?siteLanguage|g" "$HOME"/BirdNET-Pi/scripts/common.php
```

**Comment by Scraggarax** [comment]
Sorry to bring up an old thread here, but where is this line of code? I'd like to update my region, but I'm not super familiar where I'd find the file/line to edit.

**Reply by alexbelgium** [reply]
Hi, just change FR in my example to the region of your choice, then you type that in the terminal embedded in birdnet-pi. Now, when ebird opens, it will focus on stats of the region that you selected! If not happy with it or if something breaks, just hit the update button of birdnet pi and it will override this change

**Reply by blogdemoi** [reply]
Very useful, thank you. Do I have to do this after each Birdnet-Pi update?

**Reply by alexbelgium** [reply]
Yes, so you should probably set a cron job to do it automatically for you

**Comment by blogdemoi** [comment]
It would be nice to have the same for the Wikipedia page...
